unity-client/Assets/Resources/Prefabs/InputController.prefab

```diff
@@ -121,3 +121,5 @@ MonoBehaviour:
   - {fileID: 11400000, guid: 7eb660acd8e8fdb4287f537b3f17e49d, type: 2}
   - {fileID: 11400000, guid: 2ee64fe39f7539d4c9a54d793d776da5, type: 2}
   - {fileID: 11400000, guid: b9d68cd32276d4d44804d2d9325da9ca, type: 2}
+  loadingBuilderTriggerTimeActions:
+  - {fileID: 11400000, guid: 4f763868020343f469cebb00aaf01960, type: 2}
```

Comment:
I think we are going to have conflict with the shortcut branch! When merging this branch or mine we should tell the other to proceed with the merge of the prefabs

Reply:
Ok, we will be synchronized! 👍